engine/app/components/citizens_advice_components/select.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="cads-form-field<% if error? %> cads-form-field--has-error<% end %>">
+  <% if error? %><div class="cads-form-field__error-marker"></div><% end %>
+  <div class="cads-form-field__content">
+    <label class="cads-form-field__label" id="<%= label_id %>" for="<%= input_id %>">
+      <%= label %>
+      <% if optional? %>
+        <span class="cads-form-field__optional"><%= "(#{t('citizens_advice_components.input.optional')})" %></span>
+      <% end %>
+    </label>
```

Comment:
This deliberately doesn't support `page_heading` as we don't have a use-case for selects to support that.  Previously it _did_ support this but by accident as the property was shared by all `Input` sub-classes.